src/functional/ffront/func_to_foast.py

```diff
@@ -362,14 +362,14 @@ def visit_ImportFrom(self, node: ast.ImportFrom, **kwargs) -> None:
 
         return foast.ExternalImport(symbols=symbols, location=self._make_loc(node))
 
-    def visit_arguments(self, node: ast.arguments) -> list[foast.FieldSymbol]:
+    def visit_arguments(self, node: ast.arguments) -> list[foast.Symbol]:
```

Comment:
Shouldn't return a list of concrete `foast.Symbol` for the correct types (DataType ?) instead of a generic one?